### PR TITLE
ToCode Solution: תרגול תחביר

### PR DESCRIPTION
**Pull request by shaiMar**

### CODE REVIEW

File: 05_syntax_lab/01.py [hunk 1]
```diff
@@ -3,3 +3,14 @@
 the user and prints the largest one
 """
```
Comment (ynonp) [inline 05_syntax_lab/01.py:5]: Hi Shai and Welcome To The Course

File: 05_syntax_lab/01.py [hunk 2]
```diff
@@ -3,3 +3,14 @@
...
 
+largestNumber = int(0)
```
Comment (ynonp) [inline 05_syntax_lab/01.py:6]: Is 0 really the largest number you expect to read? What if a user types only negative numbers?

File: 05_syntax_lab/01.py [hunk 3]
```diff
@@ -3,3 +3,14 @@
...
+
+for n in range(10):
+    print"please enter number(%d)" %(n+1)
+    currentnumber=int(raw_input())
+    if largestNumber < currentnumber:
+        largestNumber = currentnumber
+
+
+
+print('largest number is :'),largestNumber
```
Comment (ynonp) [inline 05_syntax_lab/01.py:16]: A more pythonic print would probably use str.format:
`print 'largest number is: {}'.format(largestNumber)`

File: 05_syntax_lab/01.py (hunk 2 above)
Comment (shaiMar) [inline 05_syntax_lab/01.py:6]: so what are you suggesting?

Comment (ynonp) [inline 05_syntax_lab/01.py:6]: If all inputs are negative then your program would print 0 as the largest value (and not for instance the actual largest value from the input).

One way around this is to differentiate the first value from the rest, and use the first read value as the initial value for `largestNumber`

A second approach is to use the lowest value in the world as the initial value. In python that lowest value is `float('-inf')` (which translates to the negative infinity or מינוס אינסוף)

File: 05_syntax_lab/05.py [hunk 4]
```diff
@@ -3,3 +3,15 @@
 until it finds a number that is divisable by: 7, 13 and 15
 """
 
+from random import randint
+
+numberToDiviseto = 7*13*15
```
Comment (ynonp) [inline 05_syntax_lab/05.py:8]: 👍

File: 05_syntax_lab/05.py [hunk 5]
```diff
@@ -3,3 +3,15 @@
...
+
+
+while True:
+    num = randint(1,10000000)
+    if num % numberToDiviseto ==0:
```
Comment (ynonp) [inline 05_syntax_lab/05.py:13]: In python we prefer lowercase variable names + spaces around equality operators. So prefer `if num % number_to_diviseto == 0`

File: 05_syntax_lab/02.py [hunk 6]
```diff
@@ -4,3 +4,15 @@
 If the sum is divisable by 7, also print the word "Boom"
 """
 
+from random import randint
+
+sum = int(0)
```
Comment (ynonp) [inline 05_syntax_lab/02.py:9]: Prefer `sum = 0` (no need for int(0))

**Pull request closed without merging**